### ISSUE TITLE
about  simclr batchsize>=512?

### ISSUE THREAD

**Issue by hemo0826** [binli123/dsmil-wsi#45]
Hello, can you tell me if the batchsize must be greater than 512 when extracting image features under simclr?

**Comment by binli123** [comment]
Yes, batch size will have a significant impact on the feature quality for SimCLR training. Please check SimCLR paper for more details. https://arxiv.org/pdf/2002.05709.pdf

**Comment by Rainydu184** [comment]
what GPU does this need to run?
